packages/combo-box/test/external-filtering.test.js

```diff
@@ -0,0 +1,184 @@
+import { expect } from '@esm-bundle/chai';
+import { fire, fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import './not-animated-styles.js';
+import '../vaadin-combo-box.js';
+import { getAllItems } from './helpers.js';
+
+describe('external filtering', () => {
+  let comboBox, overlay;
+
+  function setInputValue(value) {
+    comboBox.inputElement.value = value;
+    fire(comboBox.inputElement, 'input');
+  }
+
+  function getFocusedItemIndex() {
```

Comment:
This function could be also placed in `helpers.js` to not duplicate it in both files.

Reply:
Done.